examples/1_advanced/7_status_list_2021.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::str::FromStr;
+
+use identity_iota::core::Context;
+use identity_iota::core::Timestamp;
+use identity_iota::core::Url;
+use identity_iota::credential::status_list_2021::StatusList2021;
+use identity_iota::credential::status_list_2021::StatusList2021CredentialBuilder;
+use identity_iota::credential::status_list_2021::StatusList2021Entry;
+use identity_iota::credential::status_list_2021::StatusPurpose;
+use identity_iota::credential::Credential;
+use identity_iota::credential::Issuer;
+use identity_iota::credential::JwtCredentialValidatorUtils;
+use identity_iota::credential::JwtValidationError;
+use identity_iota::credential::StatusCheck;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
```

Comment:
I suggest making the example include the entire use case, similar to other examples. Like creating DID, issuing a credential ... etc